### ISSUE TITLE
Warning in tiktok

### ISSUE THREAD

**Issue by Virtualvortex857** [yt-dlp/yt-dlp#8379]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting a bug unrelated to a specific site
- [X] I've verified that I'm running yt-dlp version **2023.10.13** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)

### Provide a description that is worded well enough to be understood

I get the following warning when downloading a tiktok video, although it downloads correctly.

[TikTok] 7291461787130547410: Downloading video feed
WARNING: [TikTok] 7291461787130547410: Unable to find video in feed; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U; trying with webpage

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[debug] Command-line config: ['-vU']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.10.13 [b634ba742] (pip)
[debug] Python 3.11.4 (CPython AMD64 64bit) - Windows-10-10.0.22621-SP0 (OpenSSL 1.1.1u  30 May 2023)
[debug] exe versions: ffmpeg 6.0-essentials_build-www.gyan.dev (setts), ffprobe 6.0-essentials_build-www.gyan.dev
[debug] Optional libraries: Cryptodome-3.18.0, brotli-1.0.9, certifi-2023.05.07, mutagen-1.46.0, sqlite3-3.42.0, websockets-11.0.3
[debug] Proxy map: {}
[debug] Loaded 1890 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Available version: stable@2023.10.13, Current version: stable@2023.10.13
yt-dlp is up to date (stable@2023.10.13)
```

**Comment by gamer191** [comment]
Can you please post a log of `yt-dlp -vU URL`

**Comment by xD1120** [comment]
[debug] Command-line config: ['-vU', 'https://vm.tiktok.com/########/']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.10.13 [b634ba742] (pip)
[debug] Python 3.11.4 (CPython AMD64 64bit) - Windows-10-10.0.22621-SP0 (OpenSSL 1.1.1u  30 May 2023)
[debug] exe versions: ffmpeg 6.0-essentials_build-www.gyan.dev (setts), ffprobe 6.0-essentials_build-www.gyan.dev
[debug] Optional libraries: Cryptodome-3.18.0, brotli-1.0.9, certifi-2023.05.07, mutagen-1.46.0, sqlite3-3.42.0, websockets-11.0.3
[debug] Proxy map: {}
[debug] Loaded 1890 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Available version: stable@2023.10.13, Current version: stable@2023.10.13
yt-dlp is up to date (stable@2023.10.13)
[vm.tiktok] Extracting URL: https://vm.tiktok.com/########/
[vm.tiktok] ZMj4G1pXo: Downloading webpage
[TikTok] Extracting URL: https://www.tiktok.com/@########/video/7291461787130547410?_d=secCgYIASAHKAESPgo8CtkTflIfY7jDbY2Ne0RKpiuNs80b%2BYQ8pTixxkKRlKgiUMPaRK3jab5w399cuLhELPVEAroPhh0ZEngUGgA%3D&_r=1&aweme_type=40&checksum=53cb13b7df5c3e4683a20b7af9f69b87553e92cfaf62942f0fe859abbfe6fe67&preview_pb=0&sec_user_id=MS4wLjABAAAAD5vVwBx4hrHKfYKYCQySedC0-1I4A-UhYDEff5CYjqh43moRoFehLhV-NJJetDGW&share_app_id=1233&share_item_id=7291461787130547461&share_link_id=849249b0-b847-45e0-90ef-8ba8c00ddfaf&sharer_language=es&social_share_type=0&source=h5_m&story_type=1&story_uid=83782077115932672&timestamp=169767734095&u_code=dc1m647d89253d&ug_btm=b5836%2Cb2878&ugbiz_name=MAIN&user_id=6817323537335124998&utm_campaign=client_share&utm_medium=android&utm_source=copy
[TikTok] 7291461787130547410: Downloading video feed
WARNING: [TikTok] 7291461787130547410: Unable to find video in feed; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U; trying with webpage
[TikTok] 7291461787130547410: Downloading webpage
[debug] Extractor gave empty title. Creating a generic title
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, hdr:12(7), vcodec:vp9.2(10), channels, acodec, size, br, asr, proto, vext, aext, hasaud, source, id
[debug] Default format spec: bestvideo*+bestaudio/best
[info] 7291461787130547410: Downloading 1 format(s): download

**Comment by bashonly** [comment]
Duplicate of #6483
